<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roomWidget</class>
 <widget class="QTabWidget" name="roomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="layerTab">
   <attribute name="title">
    <string>Layers</string>
   </attribute>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="hideButton">
       <property name="text">
        <string>Hide Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showButton">
       <property name="text">
        <string>Show Layer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="layerListWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>19</y>
      <width>310</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>High of layer :</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget">
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="title">
    <string>Rooms</string>
   </attribute>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rooms:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWalls">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_rooms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>299</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="new_room_button">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create new room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>New </string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Alt+N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_room">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete selected room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close Define Room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="statusTip">
        <string>Close widget</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+L</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add wall to selected room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Alt+A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_button">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete wall from selected room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Delete </string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Alt+R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="caption">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show room caption&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Show Caption</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Elevation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>151</width>
      <height>93</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="chname_edit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="elevation_edit"/>
     </item>
     <item>
      <widget class="QComboBox" name="classBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="highlight_all">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Highlight all rooms</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tab">
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="title">
    <string>Doors</string>
   </attribute>
   <widget class="QListWidget" name="crossingList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doors:</string>
    </property>
   </widget>
   <widget class="QLabel" name="crossing_between">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Door between</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>190</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="auto_assign_doors">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Assign Door to room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Auto Assign</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>309</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="addCrossingButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add new Door&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeCrossingButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete selected door&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton_2">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close Define Room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>309</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QComboBox" name="roomBox1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="and_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>and</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="roomBox2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_3">
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="title">
    <string>Obstacles</string>
   </attribute>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Obstacles:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>600</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWallsObs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_obstacles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>210</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>0</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="new_obs_button">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create new obstacle&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>New </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_obs">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete selected obstacle&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Delete </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton_3">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close Define Room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="statusTip">
        <string>Close widget</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="chname_edit_obs"/>
     </item>
     <item>
      <widget class="QPushButton" name="caption_obs">
       <property name="text">
        <string>Show Caption</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="is_in">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>is in:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="roomBox_obs">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="add_button_obs">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_button_obs">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>new_room_button</tabstop>
  <tabstop>list_rooms</tabstop>
  <tabstop>delete_room</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>highlight_all</tabstop>
  <tabstop>chname_edit</tabstop>
  <tabstop>elevation_edit</tabstop>
  <tabstop>classBox</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>remove_button</tabstop>
  <tabstop>caption</tabstop>
  <tabstop>listWalls</tabstop>
  <tabstop>crossingList</tabstop>
  <tabstop>auto_assign_doors</tabstop>
  <tabstop>addCrossingButton</tabstop>
  <tabstop>removeCrossingButton</tabstop>
  <tabstop>closeButton_2</tabstop>
  <tabstop>roomBox1</tabstop>
  <tabstop>roomBox2</tabstop>
  <tabstop>closeButton_4</tabstop>
  <tabstop>listWallsObs</tabstop>
  <tabstop>list_obstacles</tabstop>
  <tabstop>new_obs_button</tabstop>
  <tabstop>delete_obs</tabstop>
  <tabstop>closeButton_3</tabstop>
  <tabstop>chname_edit_obs</tabstop>
  <tabstop>caption_obs</tabstop>
  <tabstop>roomBox_obs</tabstop>
  <tabstop>add_button_obs</tabstop>
  <tabstop>remove_button_obs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
